securedrop/manage.py

```diff
@@ -180,7 +187,13 @@ def _add_user(is_admin=False):
         user = Journalist(username=username,
                           password=password,
                           is_admin=is_admin,
-                          otp_secret=otp_secret)
+                          otp_secret=otp_secret,
+                          is_totp=not is_hotp)
+        if args.password:
+            user.pw_salt = user._gen_salt()
+            user.pw_hash = user._scrypt_hash(args.password, user.pw_salt)
```

Comment:
(note the salting and hashing is done by `Journalist.set_password` in `models.py` automatically)

Reply:
The Journalist.set_password function is not used because it enforces
seven words diceware passwords and the demo password does not comply.
Although it would make for a more educative example to use a seven
words password, it is more difficult to copy/paste, all pages
displaying the existing password would have to be modified and the
users would need to update.